app/test-utils/testHarness.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
what's this one about? do we still need it

Reply:
needed it for browsersync if I remember correctly so can get binned if that is the case

Reply:
:) yes let's tidy that up.

Reply:
removed